queue/remote_ordered.go

```diff
@@ -57,21 +58,23 @@ func (q *remoteSimpleOrdered) Next(ctx context.Context) amboy.Job {
 				Start: time.Now(),
 			}
 			job.UpdateTimeInfo(ti)
-			err := q.driver.Lock(ctx, job)
```

Comment:
Why remove this?

Reply:
it just moves to line 75, after we check it...

this is also only in the ordered queue.